cms/wizards/forms.py

```diff
@@ -43,6 +45,71 @@ def optional_fields(self):
         return [f for f in self.visible_fields() if not f.field.required]
 
 
+class WizardOptionWidgets(forms.RadioSelect):
+    if DJANGO_1_10:
+        class WizardOptionRenderer(forms.widgets.RadioFieldRenderer):
+            class WizardOptionInput(forms.widgets.RadioChoiceInput):
+
+                def __init__(self, name, value, attrs, choice, index):
+                    super(WizardOptionWidgets.WizardOptionRenderer.WizardOptionInput, self).__init__(
+                        name, value, attrs, choice, index
+                    )
+                    try:
+                        wizard = wizard_pool.get_entry(choice[0])
```

Comment:
I added this because of failures in the test suite (can't recall now exactly where, though)